src/renderer/lib/hooks/useUpdateHandler.ts

```diff
@@ -0,0 +1,20 @@
+import { ipc_installUpdate } from "common/ipc";
+import { useToasts } from "react-toast-notifications";
+
+export const useUpdateHandler = () => {
```

Comment:
Likewise this should be renamed to something like: `useInstallAppUpdate` or something.

```suggestion
export const useInstallAppUpdate = () => {
```